videostore/videostore.go

```diff
@@ -28,12 +28,13 @@ const (
 	deleterInterval = 1  // minutes
 	retryInterval   = 1  // seconds
 	asyncTimeout    = 60 // seconds
-	tempPath        = "/tmp"
 
 	// TimeFormat is how we format the timestamp in output filenames and do commands.
 	TimeFormat = "2006-01-02_15-04-05"
 )
 
+var tempPath = os.TempDir()
```

Comment:
this is going to be invoked on the build machine at compile time not on the machine that the module is run on... I recommend instead replacing all previous instances of `/tmp` with `os.TempDir()` at runtime.